include/quest_log.h

```diff
@@ -15,7 +15,7 @@ struct UnkStruct_3005E90
 };
 
 extern u8 gUnknown_3005E88;
-extern u16 gUnknown_3005E8C;
+extern u16 sNumQuestLogs;
 extern struct UnkStruct_3005E90 gUnknown_3005E90;
 extern struct UnkStruct_203AE98 * gUnknown_3005E94;
```

Comment:
These things are in COMMON but are they really used outside quest_log.c?

Reply:
shrug